bedrock/mozorg/templates/mozorg/moss/index-092018.html

```diff
@@ -35,19 +35,19 @@ <h3 class="moss-subhead">{{ _('Track I: Foundational Technology') }}</h3>
       </div>
     </li>
 
-    <li>
-      <h3 class="moss-subhead">{{ _('Track II: Mission Partners') }}</h3>
+    <li id="mission-partners">
+      <h3 class="moss-subhead">{{ _('Tracks II & IV: Global Mission Partners') }}</h3>
```

Comment:
I'm not sure if any locales have translated this page yet, but this template is behind an l10n activation tag in the view: https://github.com/mozilla/bedrock/blob/master/bedrock/mozorg/views.py#L216

If this page is being actively localized, all string changes will need to be behind an l10n conditional tag. If not, we're ok to make these changes.

Perhaps @peiying2 can advise here on translation status.